DigitalLearningSolutions.Web/Views/Support/RequestSupportTicket/TypeOfRequest.cshtml

```diff
@@ -0,0 +1,65 @@
+﻿@using DigitalLearningSolutions.Data.Models.Support
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.Support.RequestSupportTicket;
+@model RequestTypeViewModel;
+@{
+    var errorHasOccurred = !ViewData.ModelState.IsValid;
+    ViewData["Title"] = "Type of Request";
+    var count = @Model.RequestTypes.Count();
+    var seperator = 0;
+}
+<link rel="stylesheet" href="@Url.Content("~/css/support/support.css")" asp-append-version="true">
+@if (errorHasOccurred)
+{
+    <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Type) })" />
+}
+<div class=" nhsuk-u-margin-top-1">
+    <a class="nhsuk-button nhsuk-button--secondary" role="button" asp-controller="RequestSupportTicket" asp-action="Index">
```

Comment:
These Back links should use the [NHS Design System back-link component](https://service-manual.nhs.uk/design-system/components/back-link), as per wireframes.

Reply:
This has been implemented and changes are pushed. Please verify.